build.gradle

```diff
@@ -1,39 +1,41 @@
 buildscript {
     ext {
-        springBootVersion = '1.5.6.RELEASE'
         coverallsVersion = '2.8.1'
     }
     repositories {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
         classpath("org.kt3k.gradle.plugin:coveralls-gradle-plugin:${coverallsVersion}")
     }
 }
 
 apply plugin: 'java'
-apply plugin: 'eclipse'
-apply plugin: 'org.springframework.boot'
 apply plugin: 'jacoco'
 apply plugin: 'com.github.kt3k.coveralls'
+apply plugin: 'maven'
 
 version = '0.0.1-SNAPSHOT'
+group = 'mertz.security'
```

Comment:
group = 'com.github.mert-z'
This will be automatically fixed for you when you do a git pull.